README.md

```diff
@@ -280,7 +280,7 @@ interface State {
 class Hello extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    this.state = { currentEnthusiasm: props.enthusiasmLevel || 1 };
+    this.state = { currentEnthusiasm: props.enthusiasmLevel == null ? 1 : props.enthusiasmLevel };

```

Comment:
Enthusiasm level cannot be `null`. It can be `number` or `undefined`. When you do a loose comparison with double equals `==`, any 'falsy' value will go to the default 1. That means `props.enthusiasmLevel == null` will be true for 0 as well, and give you the default 1, which is not the intended behavior (if it's 0 should keep 0 in the state so it throws an error on rendering).
 You can check for strict comparison `=== undefined` instead.